ddtrace/ext/git.py

```diff
@@ -158,13 +158,11 @@ def extract_user_git_metadata(env=None):
     branch = normalize_ref(env.get("DD_GIT_BRANCH"))
     tag = normalize_ref(env.get("DD_GIT_TAG"))
 
-    if env.get("DD_GIT_TAG"):
-        branch = None
-
     # if DD_GIT_BRANCH is a tag, we associate its value to TAG instead of BRANCH
     if "origin/tags" in env.get("DD_GIT_BRANCH", "") or "refs/heads/tags" in env.get("DD_GIT_BRANCH", ""):
         branch = None
-        tag = normalize_ref(env.get("DD_GIT_BRANCH"))
```

Comment:
should this just be `tag = branch`, `branch = None`? Else we unnecessarily normalize the branch twice (although that already existed)?